cmd/flow-aggregator/flow-aggregator.go

```diff
@@ -64,9 +63,7 @@ func run(configFile string) error {
 		metav1.NamespaceAll,
 		informerDefaultResync,
 		cache.Indexers{},
-		func(options *metav1.ListOptions) {
-			options.FieldSelector = fields.OneTermEqualSelector("spec.hostNetwork", "false").String()
-		},
+		nil,
```

Comment:
the informer no longer needs to be filtered, you can revert back to the old version of the code since this was introduced recently

Reply:
Sure, updated.